### PR TITLE
Adding clusterloader2 job

### PR DESCRIPTION
**Pull request by krzysied**
Adding new ci job for clusterloader2.
Adjusting previous clusterloader job, so it will use `test-cmd` flags.

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-config.yaml [hunk 1]
```diff
@@ -713,9 +713,40 @@ periodics:
       - --gcp-node-image=gci
       - --gcp-project=k8s-jkns-clusterloader
       - --gcp-zone=us-central1-f
-      - --perf-tests
       - --provider=gce
       - --test=false
+      - --test-cmd=$GOPATH/src/k8s.io/perf-tests/run-e2e.sh
+      - --test-args=cluster-loader
+      - --test-cmd-name=clusterloader
```
Comment (shyamjvs) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-config.yaml]: nit: -> ClusterLoader (as I think we're mostly using capitals for steps)

**Comment by shyamjvs** [comment]
/hold
As we need change https://github.com/kubernetes/test-infra/pull/9106 to rollout first.

File: config/jobs/kubernetes/sig-scalability/sig-scalability-config.yaml [hunk 2]
```diff
@@ -713,9 +713,40 @@ periodics:
...
+      args:
+      - --repo=k8s.io/kubernetes=master
+      - --repo=k8s.io/perf-tests=master
+      - --root=/go/src
+      - --timeout=80
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --extract=ci/latest
+      - --gcp-node-image=gci
```
Comment (shyamjvs) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-config.yaml]: I think you can remove this - it's default iirc.

File: config/jobs/kubernetes/sig-scalability/sig-scalability-config.yaml [hunk 3]
```diff
@@ -713,9 +713,40 @@ periodics:
...
+      - --gcp-nodes=3
+      - --gcp-project=k8s-jkns-clusterloader2
+      - --gcp-zone=us-central1-f
+      - --provider=gce
+      - --test=false
+      - --test-cmd=$GOPATH/src/k8s.io/perf-tests/run-e2e.sh
+      - --test-args=cluster-loader2 --testconfig=testing/density/config.yaml --nodes=3
+      - --test-cmd-name=clusterloader2
```
Comment (shyamjvs) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-config.yaml]: -> ClusterLoaderV2

File: config/jobs/kubernetes/sig-scalability/sig-scalability-config.yaml [hunk 4]
```diff
@@ -713,9 +713,40 @@ periodics:
...
+      - --repo=k8s.io/perf-tests=master
+      - --root=/go/src
+      - --timeout=80
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --extract=ci/latest
+      - --gcp-node-image=gci
+      - --gcp-nodes=3
+      - --gcp-project=k8s-jkns-clusterloader2
```
Comment (shyamjvs) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-config.yaml]: If you've created a new project manually, it'll most likely not work (as you need some authentication, service accounts, etc setup for prow to work). Either remove this and let it be auto-selected by boskos or just use the same project as the other clusterloader job (if there's enough quota there).

File: config/jobs/kubernetes/sig-scalability/sig-scalability-config.yaml (hunk 2 above)
Comment (krzysied) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-config.yaml]: line removed.

File: config/jobs/kubernetes/sig-scalability/sig-scalability-config.yaml (hunk 3 above)
Comment (krzysied) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-config.yaml]: done.

**Comment by shyamjvs** [comment]
/lgtm
/hold cancel
(As I think it's ok since we're only removing old flags in that change)

File: config/jobs/kubernetes/sig-scalability/sig-scalability-config.yaml (hunk 4 above)
Comment (krzysied) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-config.yaml]: changed to already existing project - k8s-jkns-clusterloader.

**Comment by shyamjvs** [comment]
@krzysied Seems like the job is failing due to some invalid flag - https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-perf-tests-e2e-gce-clusterloader2/1
Could you PTAL?

**Pull request merged**